apidef/api_definitions.go

```diff
@@ -802,6 +813,14 @@ func (a *APIDefinition) EncodeForDB() {
 	if a.Auth.AuthHeaderName == "" {
 		a.Auth = a.AuthConfigs["authToken"]
 	}
+	// JWTScopeToPolicyMapping and JWTScopeClaimName are deprecated and following code ensures backward compatibility
+	if a.UseOpenID && a.Scopes.OIDC.ScopeToPolicy != nil {
```

Comment:
What happens if scopeToPolicy is empty but not nil. It might better if we check claim name is “” or not.